src/assets/scully-routes.json

```diff
@@ -1 +1 @@
File":"buffer.md"},{"route":"/operators/transformation/bufferCount","sourceFile":"bufferCount.md"},{"route":"/operators/transformation/bufferTime","sourceFile":"bufferTime.md"},{"route":"/operators/transformation/bufferToggle","sourceFile":"bufferToggle.md"},{"route":"/operators/transformation/bufferWhen","sourceFile":"bufferWhen.md"},{"route":"/operators/transformation/concatMap","sourceFile":"concatMap.md"},{"route":"/operators/transformation/concatMapTo","sourceFile":"concatMapTo.md"},{"route":"/operators/transformation/exhaust","sourceFile":"exhaust.md"},{"route":"/operators/transformation/exhaustMap","sourceFile":"exhaustMap.md"},{"route":"/operators/transformation/expand","sourceFile":"expand.md"},{"route":"/operators/transformation/groupBy","sourceFile":"groupBy.md"},{"route":"/operators/transformation/map","sourceFile":"map.md"},{"route":"/operators/transformation/mapTo","sourceFile":"mapTo.md"},{"route":"/operators/transformation/mergeMap","sourceFile":"mergeMap.md"},{"route":"/operators/transformation/mergeMapTo","sourceFile":"mergeMapTo.md"},{"route":"/operators/transformation/mergeScan","sourceFile":"mergeScan.md"},{"route":"/operators/transformation/pairwise","sourceFile":"pairwise.md"},{"route":"/operators/transformation/partition","sourceFile":"partition.md"},{"route":"/operators/transformation/pluck","sourceFile":"pluck.md"},{"route":"/operators/transformation/scan","sourceFile":"scan.md"},{"route":"/operators/transformation/switchMap","sourceFile":"switchMap.md"},{"route":"/operators/transformation/switchMapTo","sourceFile":"switchMapTo.md"},{"route":"/operators/transformation/window","sourceFile":"window.md"},{"route":"/operators/transformation/windowCount","sourceFile":"windowCount.md"},{"route":"/operators/transformation/windowTime","sourceFile":"windowTime.md"},{"route":"/operators/transformation/windowToggle","sourceFile":"windowToggle.md"},{"route":"/operators/transformation/windowWhen","sourceFile":"windowWhen.md"},{"route":"/operators/utility/delay","sourceFile":"delay.md"},{"route":"/operators/utility/delayWhen","sourceFile":"delayWhen.md"},{"route":"/operators/utility/dematerialize","sourceFile":"dematerialize.md"},{"route":"/operators/utility/finalize","sourceFile":"finalize.md"},{"route":"/operators/utility/observeOn","sourceFile":"observeOn.md"},{"route":"/operators/utility/repeat","sourceFile":"repeat.md"},{"route":"/operators/utility/repeatWhen","sourceFile":"repeatWhen.md"},{"route":"/operators/utility/subscribeOn","sourceFile":"subscribeOn.md"},{"route":"/operators/utility/tap","sourceFile":"tap.md"},{"route":"/operators/utility/timeInterval","sourceFile":"timeInterval.md"},{"route":"/operators/utility/timeout","sourceFile":"timeout.md"},{"route":"/operators/utility/timeoutWith","sourceFile":"timeoutWith.md"},{"route":"/version-6/deprecations","sourceFile":"deprecations.md"},{"route":"/version-6/migration","sourceFile":"migration.md"},{"route":"/version-6/pipeable-operators","sourceFile":"pipeable-operators.md"}]
```

Comment:
This file must to be discard because it is generated by the CI

Reply:
That shouldn't have been committed either.